extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "The SQL Database Projects extension for Azure Data Studio allows users to develop and publish database schemas.",
-  "version": "0.8.0",
+  "version": "0.8.1",
   "publisher": "Microsoft",
   "preview": true,
   "engines": {
```

Comment:
also bump azdata dependency to >=1.28.0 since the dashboard is new in data workspaces in this release

Reply:
Done.